library/include/ck/library/reference_tensor_operation/cpu/reference_conv_bwd_data.hpp

```diff
@@ -72,9 +72,9 @@ struct ReferenceConvBwdData : public device::BaseOperator
             if constexpr(NumDimSpatial == 1)
             {
                 auto f_ncw = [&](auto n, auto c, auto wi) {
-                    std::size_t K  = arg.weight_.mDesc.GetLengths()[0];
-                    std::size_t X  = arg.weight_.mDesc.GetLengths()[2];
-                    std::size_t Wo = arg.output_.mDesc.GetLengths()[2];
+                    int K  = arg.weight_.mDesc.GetLengths()[0];
+                    int X  = arg.weight_.mDesc.GetLengths()[2];
+                    int Wo = arg.output_.mDesc.GetLengths()[2];
```

Comment:
Why not use std::size_t?

Reply:
[594a4c2](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/212/commits/594a4c22d129e4eac6930355ea180a0e156735e5)